internal/external-dns/registry/txt.go

```diff
@@ -480,7 +480,7 @@ func (im *TXTRegistry) removeFromCache(ep *endpoint.Endpoint) {
 	}
 }
 
-func (im *TXTRegistry) NewLabelsFromString(labelText string, aesKey []byte) (owner, version string, labels endpoint.Labels, err error) {
```

Comment:
Why was this change need?

Reply:
This is not to initialise a registry to translate targets into labels. 
I have no idea why it was a part of the registry to begin with - it uses nothing from it for its logic :thinking: